--- geeqie/doc/Makefile.am-orig	2021-12-25 20:51:18.520366689 -0500
+++ geeqie/doc/Makefile.am	2021-12-25 20:55:06.094142557 -0500
@@ -5,7 +5,7 @@
 EXTRA_DIST = docbook2html.sh docbook icons
 
 html/GuideIndex.html: docbook/GuideIndex.xml
-	rm -rf html; mkdir html; cp $(srcdir)/icons/* html/
+	rm -rf html; mkdir html; cp $(srcdir)/icons/* docbook/
 	if [ -x "$(GNOME_DOC_TOOL)" ]; then \
 		"$(GNOME_DOC_TOOL)" html -o html $(srcdir)/docbook/GuideIndex.xml ; \
 	else \
